lib/ansible/modules/network/nmcli.py

```diff
@@ -524,7 +524,8 @@ class Nmcli(object):
 
     platform='Generic'
     distribution=None
-    bus=dbus.SystemBus()
+    if HAVE_DBUS:
```

Comment:
This should be dealt with at
https://github.com/ansible/ansible/pull/19060/files#diff-fc8edeab30941b14afd491f7f42fb881R1139

Not sure why this is been run before main, that feels like an implementation issue